x-pack/plugins/cases/server/client/cases/create.test.ts

```diff
@@ -495,7 +495,7 @@ describe('create', () => {
       );
     });
 
-    it('should not throw an error and set default value when customFields are undefined', async () => {
+    it('should not throw an error and fill out missing customFields when they are undefined', async () => {
```

Comment:
nit: `update.test.ts` checks for a slightly different logic. here `customFields` is undefined as a whole, there we define one and guarantee the call is made with the missing `customField` populated. But I guess all that matters is that we guarantee `fillMissingCustomFields` is called 🤔